node/hack/query/ccqlistener/ccqlistener.go

```diff
@@ -92,6 +91,30 @@ func main() {
 	defer cancel()
 	logger, _ := zap.NewDevelopment()
 
+	if *envStr != "" {
+		// If they specify --env then use the defaults for the network parameters and don't allow them to override them.
+		if *p2pNetworkID != "" || *p2pBootstrap != "" {
+			logger.Fatal(`If "--env" is specified, "--network" and "--bootstrap" may not be specified`)
+		}
+		env, err := common.ParseEnvironment(*envStr)
+		if err != nil || (env != common.MainNet && env != common.TestNet) {
+			logger.Fatal(`Invalid value for "--env", should be "mainnet" or "testnet"`)
```

Comment:
In `ccq/query_server.go` we mentioned of `devnet`. Do we need to include `devnet` here as well? same for line 71, 100, 101

Reply:
No, `ccqlistener` does not support devnet, only mainnet and testnet.